.github/workflows/test-ubuntu.yml

```diff
@@ -0,0 +1,18 @@
+name: CI Build Ubuntu
+on:
+  schedule:
+   - cron: '0 0 * * 0'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - name: Build project with Maven
```

Comment:
Sorry for the confusion. I made a mistake. It is not version v7.0.2 but v0.7.2. 
The version of the shell checker on the Mac (installed via brew) is newer than the one on Ubuntu (already set up). Therefore, new shell errors were detected in the Mac build, which I solved via the mentioned ignore.

Reply:
Thanks for clarification - that makes sense now.
Did we investigate the shellcheck warnings before just ignoring them?
I am asking because sometimes developers see this `shellcheck disable ...` and start to think this is a general pattern to put this if an error comes up - problem solved. However, the `shellcheck disable ...` shall only be used for false positives... Maybe the newer version of shellcheck is correct and our code is bad and needs improvement rather than adding `shellcheck disable ...` .

Reply:
The only new error message would be the following: https://github.com/alfeilex/ide/runs/7406568486?check_suite_focus=true#step:5:861

Shellcheck: [SC1091](https://www.shellcheck.net/wiki/SC1091)

Unfortunately, I cannot evaluate if it is bad implementation.